.github/workflows/ci.yml

```
@@ -0,0 +1,158 @@
+name: CI
+
+on: [push]
+
+env:
+  # Would be more usual to set R_LIBS_USER, but R uses R_LIBS first if present
+  # ...and it's always present here, because the rocker/tidyverse base image
+  # checks at R startup time for R_LIBS and R_LIBS_USER, sets both if not found
+  R_LIBS: ~/R/library
+  _R_CHECK_LENGTH_1_CONDITION_: true
+  _R_CHECK_LENGTH_1_LOGIC2_: true
+  # Avoid compilation check warnings that come from the system Makevars, not us
+  # See https://stat.ethz.ch/pipermail/r-package-devel/2019q2/003898.html
+  _R_CHECK_COMPILATION_FLAGS_KNOWN_: -Wformat -Werror=format-security -Wdate-time
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    container: infotroph/pecan-depends:latest
+    steps:
+    - uses: meetup/express-checkout@master #clones with depth 1; skips tags, branches, submodules
```

@infotroph I will suggest that we can use `actions/checkout@v2` because the v2 has made big leaps and is now comparably fast compared to meetup/express-checkout. You can read about it [here](https://github.com/meetup/express-checkout).
```suggestion
    - uses: actions/checkout@v2 #clones with depth 1; skips tags, branches, submodules
```